pom.xml

```diff
@@ -176,7 +176,7 @@ under the License.
     <plexusInterpolationVersion>1.26</plexusInterpolationVersion>
     <plexusTestingVersion>1.0.0</plexusTestingVersion>
     <plexusXmlVersion>4.0.1</plexusXmlVersion>
-    <resolverVersion>2.0.0-alpha-5</resolverVersion>
+    <resolverVersion>2.0.0-alpha-6</resolverVersion>
```

Comment:
Should that be in a separate commit for clarity and release notes ?

Reply:
There is a separate PR just for the update https://github.com/apache/maven/pull/1373 (and removal of some deprecation warnings in tests)